client/react/components/chat/channelBar.jsx

```diff
@@ -0,0 +1,32 @@
+import React, {Component} from 'react';
+import { connect } from 'react-redux';
+import { push } from 'react-router-redux';
+import { Container,Icon, Menu} from 'semantic-ui-react';
+
+import './styles.css';
+
+const listChannels = ({
```

Comment:
Put this inside the main render method we we don't need to pass in all these variables.

We are going to try and have logic in the render method before the return.

This means the render method can't have an implicit return.